OCDiffTests/OCDAPIComparatorTests.m

```diff
@@ -1171,7 +1189,7 @@ - (void)testAddRemoveForName:(NSString *)name base:(NSString *)base addition:(NS
     XCTAssertEqualObjects(differences, @[], @"Unchanged test failed for %@", name);
 
     // Unchanged, different line number
-    differences = [self differencesBetweenOldSource:addition newSource:[@"\n" stringByAppendingString:addition]];
+    differences = [self differencesBetweenOldSource:addition newSource:[@"" stringByAppendingString:addition]];
```

Comment:
The newline is an essential part of this test so this needs to be put back. Where possible objc-diff uses the Clang-defined Unified Symbol Resolution (USR) as the key on which to determine if an API has been added or removed. For declarations that are not externally visible Clang includes location information in the USR if the declaration is not in a system header (system headers are considered to be sane). Unfortunately this means that the USR cannot be used as a key for these entity types as simply moving them to a different line would look like a removal an addition to objc-diff. For such cases [`keyForCursor:`](https://github.com/mattstevens/objc-diff/blob/95998186066dc66e50e291605d011d8b6a4d4d52/OCDiff/OCDAPIComparator.m#L147) falls back to generating and objc-diff-specific key.